src/wh_client_cryptocb.c

```diff
@@ -84,7 +85,7 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
                 /* read out */
                 else {
                     info->pk.curve25519kg.key->devCtx =
-                        (void*)0 + packet->pkCurve25519kgRes.keyId;
```

Comment:
I think I found a better way to do this that makes all sizes the same when I added ecc
```suggestion
                        XMEMCPY((void*)&info->pk.curve25519kg.key->devCtx, (void*)&packet->pkCurve25519kgRes.keyId, sizeof(packet->pkCurve25519kgRes.keyId));
```

Reply:
I couldn't find the definition for WOLFHSM_PACKET_STUB_SIZE in any of your PR's.  Can you point that to me?

Reply:
@billphipps [defined here](https://github.com/wolfSSL/wolfHSM/blob/b92437a951a813572f2649a9020cce64fe61cade/wolfhsm/wh_packet.h#L36)